### PR TITLE
Fix automation re: master->main rename

### LINKED ISSUE #1282: Please rename the master branch to the main branch.

### PR DESCRIPTION
**Pull request by cevich**
Ref: https://github.com/containers/image/pull/1272 and https://github.com/containers/skopeo/issues/1282

Fixes #1282

### CODE REVIEW

**Comment by cevich** [comment]
Oof, [the Dockerfile build](https://quay.io/repository/skopeo/ci/build/7d2d1016-77fd-438e-948d-f8e11281c5d9) is currently broken as well, gunna needa fix that too...

**Comment by cevich** [comment]
...okay, local testing shows the Dockerfile now builds successfully.  I added a notification to the auto-builder (on quay.io) for this image: It should now post future build-failure notices to our podman-monitor list.

**Comment by mtrmac** [comment]
> Oof, the Dockerfile build is currently broken as well, gunna needa fix that too...

To be explicit, those are the `GO111MODULE` changes, which are otherwise unrelated?

---

The `cross` task is failing for what seems to be a Brew server failure, I have triggered a re-run.

File: Dockerfile [hunk 1]
```diff
@@ -34,6 +35,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 RUN set -x \
+    && export GO111MODULE=off \
```
Comment (mtrmac) [inline Dockerfile:38]: A nit: Could you indent the new lines the same way as the rest, please?

Comment (cevich) [reply Dockerfile:38]: Oh woops, my mistake, yep I got it...

**Comment by cevich** [comment]
@mtrmac PTAL, I fixed the whitespace and bumped up the timeout a bit to see if that helps the cross task.

**Review by mtrmac — APPROVED** [review]
Thanks! LGTM.

(I’m not sure the timeout is desirable, this just seems to be a Brew infrastructure issue where we instead of pre-built binaries burn CPU time re-building locally. But even if it gets CI unblocked for a week and then we can revert, that’s valuable for that week.)

**Comment by mtrmac** [comment]
@cevich Looking at https://github.com/Homebrew/brew/issues/11119 or https://github.com/Homebrew/brew/pull/11366 , the `brew` failure seems not to be a temporary situation (although https://status.bintray.com does currently report a scheduled maintenance).

Is the `brew` version in the macOS environment new enough, or could it be updated? If not, a `brew update` before package installation might well pay off in time.

**Comment by cevich** [comment]
> But even if it gets CI unblocked for a week and then we can revert

I'm good either way.  The same key-name can be added to individual tasks for more specific/specialized timeout values, I only stuck in a global one out of sheer laziness.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by mtrmac** [comment]
I have filed a completely uninformed #1329 with changes from this PR + `brew update`, just to see what happens.

**Comment by cevich** [comment]
> To be explicit, those are the GO111MODULE changes, which are otherwise unrelated?

Yes, mostly unrelated.  Though auto-builder on quay was specifically targeting the master branch.  When I fixed that, I noticed the image build has been failing for months, so, yeah, :disappointed: Anyway, I also added some monitoring, so we'll now get e-mails to the podman-monitor list any time it breaks in the future.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**